fastplotlib/layouts/_plot.py

```diff
@@ -45,7 +45,7 @@ def __init__(
             passed to Subplot, for example ``name``
 
         """
-        super(Plot, self).__init__(
+        super().__init__(
```

Comment:
What exactly happens with `super` calls in these situations? Because we want to pass these args to `Subplot` and `Frame` is instantiated separately with different args

Reply:
Using `super()` is equivalent to `super(CurrentClass, self)`. I double-checked that this is the case for all changes in this PR.

It will take the next class in the mro, which is `SubPlot` in this case. You can check the resolution order using `Plot.mro()`. With multiple inheritance, the classes on the right are higher up the tree.